apps/prairielearn/src/lib/question-render.ts

```diff
@@ -593,38 +588,41 @@ export async function getAndRenderVariant(
  */
 export async function renderPanelsForSubmission({
   submission_id,
-  question_id,
-  instance_question_id,
+  question,
+  instance_question,
   variant_id,
-  user_id,
+  user,
   urlPrefix,
   questionContext,
   authorizedEdit,
   renderScorePanels,
 }: {
```

Comment:
With what is left in this function, wouldn't it make more sense to just pass along the whole res.locals object as argument?

Reply:
Absolutely not! We're actively moving in the other direction and working to always pass along specific properties instead of `res.locals` as a whole.